transforms.go

```diff
@@ -191,10 +192,10 @@ func filterIp(ip string, captureIp captureIp) string {
 // method, the causes will be traversed until nil.
 func errorBody(configuration configuration, err error, skip int) (map[string]interface{}, string) {
 	var parent error
-	traceChain := []map[string]interface{}{}
+	var traceChain []map[string]interface{}
```

Comment:
Why this change?

Reply:
Sorry it was just a reflex to change that due to my IDE complaining about declaring an empty slice with a literal. I believe it's considered a good convention to use nil declarations (https://stackoverflow.com/a/29164565/998919), but by all means, I can revert it if this project prefers the other (previous) style.

Reply:
I guess that we always append something to it so this change is okay. The one difference is that for JSON serialization the nil slice becomes `null` and the empty slice becomes `[]` and the backend will treat those differently.

Reply:
True. I did pay attention to this in at least one other spot, to make sure we get `[]` in the JSON. Hmm I do feel now that the benefit of leaving that line as-is (e.g. what it originally was) outweighs the benefit of not allocating it (since we do "always" append to it, assuming no errors occur). I'm thinking if the code changes down the line it might become a frustrating error in case a nil slice slips through to the JSON marshaling. I'm gonna go ahead and revert it if that's fine.